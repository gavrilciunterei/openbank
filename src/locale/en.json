{
  "general": {
    "next": "Next",
    "cancel": "Cancel",
    "create_password": "Create your Password Manager",
    "es": "Spanish",
    "en": "English"
  },
  "product_information": {
    "create_password": "Create your Password Manager",
    "text_save_password": "Store all your passwords, data or any information here, forget paper notes and unprotected applications.",
    "text_master_password": "Create your master key: only you can access your secrets with it.",
    "how_does_word": "How does it work",
    "text_first_place": "First of all, you need to create a different password for your electronic belongings. You will not be able to recover your password, so remember it well.",
    "what_data_save": "What data can you save",
    "text_what_data_save": "For example, your card number, the PIN and PUK of your mobile phone, the serial number of one of your devices or any information you need to have in a safe place.",
    "data_protection_policy": "I am of legal age and I accept the processing of data according to the data protection policy."
  },
  "form": {
    "first_step": "First of all, you need to create a different password for your electronic belongings.",
    "first_step_secound": "You will not be able to recover your password, so remember it well.",
    "no_password": "No password provided.",
    "password_short": "Password is too short; should be 8 characters minimum.",
    "password_max": "Password is too long; should be 24 characters max.",
    "password_num_upper": "Password must have one uppercase and one number",
    "password_match": "Passwords must match",
    "create_master_pass": "Create your Master Password",
    "write_password": "Enter your password",
    "repeat_master_password": "Repeat your Master Password",
    "repeat_password": "Repeat your password",
    "secound_step": "You can also create a hint to help you remember your master password.",
    "create_track": "Create your hint to remember your password (optional)",
    "insert_trak": "Enter your hint",
    "track_max": "The hint can only have a maximum of 60 characters",
    "repeat_required": "Repeat password is required"
  }
}
