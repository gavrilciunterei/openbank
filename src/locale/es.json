{
  "general": {
    "next": "Siguiente",
    "cancel": "Cancelar",
    "create_password": "Crea tu Password Manager"
  },
  "product_information": {
    "text_save_password": "Guarda aquí todas tus contraseñas, datos o cualquier información, olvida las notas de papel y las aplicaciones no protegidas.",
    "text_master_password": "Crea tu clave maestra: solo tú podrás acceder a tus secretos con ella.",
    "how_does_word": "Cómo funciona",
    "text_first_place": "En primer lugar, debes crear una contraseña diferente para sus pertenencias electrónicas. No podrás recuperar tu contraseña, así que recúerdala bien.",
    "what_data_save": "Qué datos puedes guardar",
    "text_what_data_save": "Por ejemplo, el número de tu tarjeta, el PIN y el PUK de tu teléfono móvil, el número de serie de alguno de tus dispositivos o cualquier información que necesites tener en lugar seguro.",
    "data_protection_policy": "Soy mayor de edad y acepto el tratamiento de datos según la política de protección de datos."
  },
  "form": {
    "first_step": "En primer lugar, debes crear una contraseña diferente para sus pertenencias electrónicas.",
    "first_step_secound": "No podrás recuperar tu contraseña, así que recuérdala bien.",
    "no_password": "No se proporcionó contraseña.",
    "password_short": "La contraseña es demasiado corta; debe tener un mínimo de 8 caracteres.",
    "password_max": "La contraseña es demasiado larga; debe tener un máximo de 24 caracteres.",
    "password_num_upper": "La contraseña debe tener una mayúscula y un número",
    "password_match": "Las contraseñas deben coincidir",
    "create_master_pass": "Crea tu Contraseña Maestra",
    "write_password": "Escribe tu contraseña",
    "repeat_master_password": "Repite tu Contraseña Maestra",
    "repeat_password": "Repite tu contraseña",
    "secound_step": "También puedes crear una pista que te ayude a recordar tu contraseña maestra.",
    "create_track": "Crea tu pista para recordar tu contraseña (opcional)",
    "insert_trak": "Introduce tu pista",
    "track_max": "La pista solo puede tener un máximo de 60 caracteres",
    "repeat_required": "Repetir la contraseña es obligatorio"
  }
}
