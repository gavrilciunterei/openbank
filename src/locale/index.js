import i18n from 'i18next';
import { initReactI18next } from 'react-i18next';
import LanguageDetector from 'i18next-browser-languagedetector';
import es from './es.json';
import en from './en.json';

export default i18n
  .use(initReactI18next)
  .use(LanguageDetector)
  .init({
    supportedLngs: ['en', 'es'],
    resources: {
      en: {
        translation: en,
      },
      es: {
        translation: es,
      },
    },
    fallbackLng: 'en',

    detection: {
      order: [
        'cookie',
        'navigator',
        'htmlTag',
        'localStorage',
        'querystring',
        'sessionStorage',
        'path',
        'subdomain',
      ],
      caches: ['cookie'],
    },
  });
