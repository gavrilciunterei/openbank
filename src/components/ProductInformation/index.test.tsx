import React from 'react';
import renderer from 'react-test-renderer';
import { FirstInfo, SecoundInfo } from './index';
import '../../locale/index';

it('Test FirstInfo', () => {
  const tree = renderer.create(<FirstInfo />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('Test SecoundInfo', () => {
  const tree = renderer.create(<SecoundInfo />).toJSON();
  expect(tree).toMatchSnapshot();
});
