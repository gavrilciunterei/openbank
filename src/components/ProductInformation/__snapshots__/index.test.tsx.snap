// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test FirstInfo 1`] = `
<div
  className="sc-bdvvtL dRonou"
>
  <div
    className="sc-gsDKAQ iCkZym"
  >
    <img
      alt="img1"
      className="sc-dkPtRN iMZzcL"
      src="group.svg"
    />
    <p
      className="sc-hKwDye bgBOwh"
    >
      Store all your passwords, data or any information here, forget paper notes and unprotected applications.
    </p>
  </div>
  <div
    className="sc-gsDKAQ iCkZym"
  >
    <img
      alt="img2"
      className="sc-dkPtRN iMZzcL"
      src="group-3.svg"
    />
    <p
      className="sc-hKwDye bgBOwh"
    >
      Create your master key: only you can access your secrets with it.
    </p>
  </div>
</div>
`;

exports[`Test SecoundInfo 1`] = `
Array [
  <div
    className="sc-bdvvtL cYfyVu"
  >
    <p
      className="sc-hKwDye lhQqIX"
      fontWeight={500}
      size="18px"
    >
      How does it work
    </p>
    <p
      className="sc-hKwDye gdvmyK"
    >
       
      First of all, you need to create a different password for your electronic belongings. You will not be able to recover your password, so remember it well.
    </p>
  </div>,
  <div
    className="sc-bdvvtL cYfyVu"
  >
    <p
      className="sc-hKwDye lhQqIX"
      fontWeight={500}
      size="18px"
    >
      What data can you save
    </p>
    <p
      className="sc-hKwDye gdvmyK"
    >
       
      For example, your card number, the PIN and PUK of your mobile phone, the serial number of one of your devices or any information you need to have in a safe place.
    </p>
  </div>,
]
`;
